apps/google-analytics-4/frontend/src/hooks/useSidebarSlug/useSidebarSlug.spec.tsx

```diff
@@ -1,14 +1,16 @@
+// @ts-nocheck
```

Comment:
do we still need this? or maybe more line-specific rules?

Reply:
Yeah I meant to revisit that good call out. Vite's `importActual` method that replaces the jest version returns a promise, which had its typing incompatible with our SDK mocking. This test file had that error in like 5 places. I typed the block as `any` type and left a comment about it because I'm not totally sure how to resolve without fully rewriting the tests and I don't understand what we're trying to test well enough to do that with any confidence.